agent/rpc/middleware/interceptors.go

```diff
@@ -22,27 +23,26 @@ const RPCTypeInternal = "internal"
 const RPCTypeNetRPC = "net/rpc"
 
 var metricRPCRequest = []string{"rpc", "server", "call"}
-var requestLogName = "rpc.server.request"
+var requestLogName = strings.Join(metricRPCRequest, "_")
```

Comment:
`logging/named.go` seems to encourage `_` in our loggers names.

Reply:
👍